src/components/animations/Animation04/WaveCanvasBackground.module.css

```
@@ -0,0 +1,34 @@
+  width: 100%;
+  height: 100%;
+  z-index: 1;
+  pointer-events: none;
+  will-change: auto;
+}
+
+/* Optimize for different screen densities */
+@media (min-resolution: 2dppx) {
+  .waveCanvas {
+    image-rendering: -webkit-optimize-contrast;
+    image-rendering: crisp-edges;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

image-rendering is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)